src/sync/incomingSyncUtils.js

```diff
@@ -786,6 +786,7 @@ export const createOrUpdateRecord = (database, settings, recordType, record) =>
         subtotal: parseFloat(record.subtotal),
         user1: record.user1,
         paymentType: database.getOrCreate('PaymentType', record.paymentTypeID),
+        isCancellation: parseBoolean(record.is_cancellaton),
```

Comment:
the spelling error is causing a bug for me :)

```suggestion
        isCancellation: parseBoolean(record.is_cancellation),
```

Reply:
wow I think this is really great work 


ad0086e